packages/compiler-cli/src/ngtsc/diagnostics/src/error_code.ts

```diff
@@ -52,6 +52,11 @@ export enum ErrorCode {
 
   SYMBOL_NOT_EXPORTED = 3001,
   SYMBOL_EXPORTED_UNDER_DIFFERENT_NAME = 3002,
+  /**
+     Raised when a relationship between directives and/or pipes would cause a cyclic import to be
+     created that cannot be handled, such as in partial compilation mode.
+   */
```

Comment:
```suggestion
  /**
   *  Raised when a relationship between directives and/or pipes would cause a cyclic import to be
   * created that cannot be handled, such as in partial compilation mode.
   */
```
formatting nit: I think this should have `*`s on each line. I've seen documentation generate incorrectly without them.

Reply:
Indeed!!